conda-recipe/meta.yaml

```
@@ -27,29 +28,41 @@ requirements:
         - dpcpp-cpp-rt >={{ required_compiler_version }},!={{ excluded_compiler_version1 }},!={{ excluded_compiler_version2 }},!={{ excluded_compiler_version3 }}  # [win]
         - dpcpp-cpp-rt >={{ required_compiler_version }},!={{ excluded_compiler_version1 }},!={{ excluded_compiler_version2 }}  # [linux]
         - python
-        - setuptools >=63.*
-        - scikit-build >=0.15*
-        - ninja # [not win]
-        - cmake >=3.26*
-        - numba >=0.59*
-        - dpctl >=0.16*
-        - dpnp >=0.14*
-        - dpcpp-llvm-spirv
-        - wheel
+        - setuptools >=69
+        - scikit-build >=0.17
+        - ninja >=1.11.1 # [not win]
+        - cmake >=3.29
+        - numba >=0.59
+        - llvmlite >=0.42.0
+        - dpctl >=0.16
```

```suggestion
        - dpctl >0.16
```
If the code path to support version 0.16 is removed from `dpctl_iface._herlper.py`, then latest numba-dpex will no longer be compatible with 0.16 and lower versions of dpctl.

---

Yes, that's right. I've thought those changes will be included in  0.16.1. If that is true then `dpctl>0.16.1` must take place. Please confirm what is the desired version.